runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -58,10 +55,12 @@ public final class LogReplicationUtils {
 
     // Prefix of the name of queue as it will appear on the receiver after replicated.  The suffix will be the Sender
     // (Source) cluster id
-    public static final String REPLICATED_QUEUE_NAME_PREFIX = "LRQ_Recv_";
+    // Receiving queues per client name
+    public static final String REPLICATED_QUEUE_NAME_PREFIX = "LRQ_Recv_<client_name>_<source_id>_";
 
     // Stream tag applied to the replicated queue on the receiver
-    public static final String REPLICATED_QUEUE_TAG = "lrq_recv";
+    // stream tag per client name
+    public static final String REPLICATED_QUEUE_TAG = "lrq_recv_<client_name>";
```

Comment:
same here.  Also, please consider renaming the variable as 'REPLICATED_QUEUE_TAG_PREFIX'